src/SSHDebugPS/VsOutputWindowWrapper.cs

```diff
@@ -0,0 +1,138 @@
+        }
+
+        private const string DefaultOutputPane = "Debug";
+
+        private static Dictionary<string, PaneInfo> panes = new Dictionary<string, PaneInfo>()
+        {
+            // The 'Debug' pane exists by default
+            { DefaultOutputPane, new PaneInfo(VSConstants.GUID_OutWindowDebugPane) }
+        };
+
+        /// <summary>
+        /// Writes text directly to the VS Output window.
+        /// </summary>
+        public static void Write(string message, string pane = DefaultOutputPane)
+        {
+            ThreadHelper.JoinableTaskFactory.RunAsync(async delegate
+            {
+                await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+                try
+                {
+                    // Get the Output window
+                    IVsOutputWindow outputWindow = outputWindowLazy.Value;
+                    if (outputWindow == null)
+                    {
+                        return;
+                    }
+
+                    // Get the pane guid
+                    PaneInfo paneInfo;
+                    if (!panes.TryGetValue(pane, out paneInfo))
+                    {
+                        // Pane didn't exist, create it
+                        paneInfo = new PaneInfo(Guid.NewGuid());
+                        panes.Add(pane, paneInfo);
+                    }
+
+                    // Get the pane
+                    IVsOutputWindowPane outputPane;
+                    if (outputWindow.GetPane(ref paneInfo.paneId, out outputPane) != VSConstants.S_OK)
+                    {
+                        // Failed to get the pane - might need to create it first
+                        outputWindow.CreatePane(ref paneInfo.paneId, pane, fInitVisible: 1, fClearWithSolution: 1);
+                        outputWindow.GetPane(ref paneInfo.paneId, out outputPane);
+                    }
+
+                    // The first time we output text to a pane, ensure it's visible
+                    if (!paneInfo.Shown)
+                    {
+                        paneInfo.Shown = true;
+
+                        // Switch to the pane of the Output window
+                        outputPane.Activate();
+
+                        // Show the output window
+                        IVsUIShell shell = shellLazy.Value;
+                        if (shell != null)
+                        {
+                            object inputVariant = null;
+                            shell.PostExecCommand(VSConstants.GUID_VSStandardCommandSet97, (uint)VSConstants.VSStd97CmdID.OutputWindow, 0, ref inputVariant);
+                        }
+                    }
+
+                    // Output the text
+                    outputPane.OutputString(message);
+                }
+                catch (Exception)
+                {
+                    Debug.Fail("Failed to write to output pane.");
+                }
+            }).FileAndForget("vs/SSHDebugPS/VsOutputWindowWrapper/Write");
```

Comment:
>vs [](start = 30, length = 2)

I think you want to use the same namespace as the rest of our telemetry - "VS/Diagnostics/Debugger/SSHDebugPS/VsOutputWindowWrapper/Write"